### PR TITLE
[Preact]

### PR DESCRIPTION
**Pull request by octavioturra**
**Extern:** Generated.

### CODE REVIEW

File: preact/build.boot [hunk 1]
```diff
@@ -0,0 +1,31 @@
+
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.5.2"  :scope "test"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all])
+
+(def +lib-version+ "6.0.0")
+(def +version+ (str +lib-version+ "-beta"))
```
Comment (martinklepsch) [inline preact/build.boot]: please use `-0` instead of `-beta` as suffix.

File: preact/build.boot [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+
+(deftask package []
+  (comp
+    (download :url (str "https://cdnjs.cloudflare.com/ajax/libs/preact/" +lib-version+ "/preact.js")
+              :checksum "03d2993b692643a21b0fc25cae936533")
+    (download :url (str "https://cdnjs.cloudflare.com/ajax/libs/preact/" +lib-version+ "/preact.min.js")
+              :checksum "23d3be42e4b4c25b76f7ac88f976eccb")
+    (sift :move {#"preact\.js" "cljsjs/preact/development/preact.inc.js"
+                 #"preact\.min\.js" "cljsjs/preact/development/preact.min.inc.js"})
```
Comment (martinklepsch) [inline preact/build.boot]: this should be `cljsjs/preact/production/preact.min.inc.js` 👍

**Comment by martinklepsch** [comment]
Hey @octaviogb 👋  I left a few comments where things could be improved. Thanks for your work  getting Preact into CLJSJS!

**Comment by skrat** [comment]
@octaviogb Could you push the change that @martinklepsch proposed so this can be merged?

**Comment by Deraen** [comment]
Someone else can also finish this by cloning the existing repo:

```
git checkout -b octavioturra-preact master
git pull https://github.com/octavioturra/packages.git preact
```

and implementing the changes and making a new PR.

**Pull request merged**